proto/dwtools/abase/l0.test/Long.test.s

```diff
@@ -13427,14 +13427,80 @@ function longFromRange( test )
   });
 
 };
+
 //
 
-function longFromRange1( test )
+function longFromRangeWithStep( test )
 {
+  test.case = 'equal numders of range';
+  var got = _.longFromRangeWithStep( [ 1, 1 ], 2 );
+  var expected = [ ];
+  test.identical( got, expected );
 
-  console.log('Hello world');
+  test.case = 'valid range with float step';
```

Comment:
Description of test case should describe content of test case. 
What is valid range? 

What do you mean about this description?
```js
test.case = 'range[ 0 ] < range[ 1], float step';

Reply:
Ok, I get it, yes it looks better.